docs/apache-airflow-providers-amazon/auth-manager/setup/identity-center.rst

```diff
@@ -0,0 +1,142 @@
+:ref:`created manually <identity_center_manual_configuration>`.
+
+If the error message below is raised, please create the AWS IAM Identity Center application through the console
+following :ref:`these instructions <identity_center_manual_configuration_application>`. ::
+
+   Creation of SAML applications is only supported in AWS console today. Please create the application through the console.
+
+.. _identity_center_manual_configuration:
+
+Create resources manually
+-------------------------
+
+Create the instance
+~~~~~~~~~~~~~~~~~~~
+
+Please follow `AWS documentation <https://docs.aws.amazon.com/singlesignon/latest/userguide/identity-center-instances.html>`_
+to create the AWS IAM Identity Center instance.
+
+.. _identity_center_manual_configuration_application:
+
+Create the application
+~~~~~~~~~~~~~~~~~~~~~~
+
+Please follow the instructions below to create the AWS IAM Identity Center application.
+
+1. Open the `IAM Identity Center console <https://console.aws.amazon.com/singlesignon>`_.
+2. Choose **Applications**.
+3. Choose the **Customer managed** tab.
+4. Choose **Add application**.
+5. On the **Select application type** page, under **Setup preference**, choose **I have an application I want to set up**.
+6. Under **Application type**, choose **SAML 2.0**.
+7. Choose **Next**.
+8. On the **Configure application** page, under **Configure application**, enter a **Display name** for the application, such as ``Airflow``. Then, enter a Description.
+9. Under **IAM Identity Center metadata**, copy the address of the **IAM Identity Center SAML metadata file**.
+
+.. note::
+
+  You will need to set this address in Airflow configuration later.
+
+10. Under **Application metadata**, choose **Manually type your metadata values**. Then, provide the **Application ACS URL** and **Application SAML audience** as follows:
+
+   * **Application ACS URL**: ``<airflow_endpoint>/login_callback``
+   * **Application SAML audience**: ``<airflow_endpoint>/login_metadata``
+
+11. Choose **Submit**. The application is now created.
+
+Attribute mappings configuration
+================================
+
+Once the application created, you need to configure the attribute mappings.
+
+1. Go to the details page of the application that you just created.
+2. Choose **Actions**.
+3. Under **Actions**, choose **Edit attribute mappings**. You're taken to the attribute mappings configuration page.
+4. On the **Attribute mappings** page, you need to configure the different attribute mappings between your identity
+   provider and AWS IAM Identity Center. For more information on attribute mappings, see the
+   `IAM Identity Center documentation <https://docs.aws.amazon.com/singlesignon/latest/userguide/attributemappingsconcept.html>`_.
+   The AWS auth manager needs two attributes: **id** and **groups**.
+   If you use the default Identity Center directory as identity source, you can use the configuration below:
```

Comment:
Should you have documentation on using other sources? What other sources can you use?

Reply:
The number of different sources can be really big, it is hard/impossible to cover all of them. Though, I could add one of two very popular ones

Reply:
Fair enough, yeah maybe just a couple of the big ones. But they don't need full documentation. Also how it is now is fine too :+1:

Reply:
I'll keep it as is because the documentation I linked before in the documentation should help users to figure out their mappings. The documentation takes multiple different examples such as Active Directory